shell-completion/zsh/_udevadm

```diff
@@ -32,6 +32,7 @@ _udevadm_trigger(){
         '--tag-match=property[Trigger events for devices with a matching tag.]' \
         '--sysname-match=[Trigger events for devices with a matching sys device name.]' \
         '--parent-match=[Trigger events for all children of a given device.]'
+        '--uuid[Print synthetic uevent UUID.]'
```

Comment:
Hmm, previous lines have `\` at the end. Please add it here too.

Reply:
Ouch. Fixed.